doc/guides/docs/index.rst

```diff
@@ -55,6 +55,7 @@ Installing the documentation processors
 Our documentation processing has been tested to run with:
 
 * Doxygen version 1.8.13
+* Graphviz 2.43
```

Comment:
This is good, but I think you forgot to update the `apt-get` etc. lines below.

Reply:
good catch, fixed